test/reducedim.jl

```diff
@@ -30,6 +33,7 @@ safe_minabs(A::Array{T}, region) where {T} = safe_mapslices(minimum, abs.(A), re
     @test sum!(abs2, r, Areduc) ≈ safe_sumabs2(Areduc, region)
     @test maximum!(abs, r, Areduc) ≈ safe_maxabs(Areduc, region)
     @test minimum!(abs, r, Areduc) ≈ safe_minabs(Areduc, region)
+    @test count!(abs, r, Breduc) ≈ safe_count(Breduc, region)
```

Comment:
I think it's better to try something that is not an identity function. Something like

```suggestion
    @test count!(!, r, Breduc) ≈ safe_count(.!(Breduc), region)
```

Reply:
thanks, fixed